api/tests.edn

```diff
@@ -1,10 +1,10 @@
 #kaocha/v1
     {:tests [{:id           :integration
-              :source-paths ["src"]
+              :source-paths ["../derived/api/resources" "resources" "src"]
```

Comment:
The fact that this classpath ordering thing appears throughout the repo I think further supports us indicating a bit of why this is extremely important